guardowl/protocols.py

```diff
@@ -874,3 +755,177 @@ def run_DOF_scan(
         raise NotImplementedError
     elif DOF_definition["torsion"]:
         raise NotImplementedError
+
+
+def run_detect_minimum_test(
+    nnp: str,
+    implementation: str,
+    platform: Platform,
+    output_folder: str,
+    percentage: int = 10,
+    only_molecules_below_10_heavy_atoms: bool = False,
+) -> Dict[str, Tuple[float, float]]:
+    """
+    Perform a minimization on a selected compound.
+    :param nnp: The neural network potential to use.
+    :param implementation: The implementation to use.
+    :param name: The name of the molecule to simulation.
+    """
+    from guardowl.testsystems import SmallMoleculeTestsystemFactory
+    import mdtraj as md
+    from .utils import (
+        extract_drugbank_tar_gz,
+        _generate_file_list_for_minimization_test,
+        _generate_input_for_minimization_test,
+    )
+
+    # extract drugbank tar.gz file
+    extract_drugbank_tar_gz()
+
+    # generate all relevenat input files
+    files = _generate_file_list_for_minimization_test(shuffel=True)
+
+    # calculate the number of molecules to test
+    nr_of_molecules = files["total_number_of_systems"]
+    nr_of_molecules_to_test = int(nr_of_molecules * (percentage / 100))
+
+    score = {}
+    counter = 0
+
+    print(
```

Comment:
log.info

Reply:
moved to log.info for all print statments